src/application/modules/books/use-cases/remove-book/remove-book-use-case.ts

```diff
@@ -0,0 +1,31 @@
+import { InternalServerHTTPError } from "@/application/shared/http/errors/internal-server-http-error";
+import { NotFoundHTTPError } from "@/application/shared/http/errors/not-found-http-error";
+import { IUseCase } from "@/application/shared/http/interfaces/use-case";
+import { BookRepository } from "../../repositories/book-repository";
+
+type IInput = {
+  bookId: string;
+};
```

Comment:
Geralmente quando é assim a gente costuma usar só interface, type é pra casos mais específicos
```suggestion
interface IInput {
  bookId: string;
};
```